### PR TITLE
Mindshield implants resist mind control attempts

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
rscadd: Sevtug's Mind Control is now prevented by mindshield implants,
but the implant is destroyed in the process. Previously implanted
personnel are encouraged to be reissued an implant to continue to resist
psychic attacks.
/:cl:
- improve don't remove

### CODE REVIEW

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 1]
```diff
@@ -1058,6 +1058,26 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
...
+			M << "<span class='sevtug'>Lbhe anabobgf jvyy abg cebgrpg lbh \
+				sberire...</span>\n\
+				<span class='userdanger'>You feel like something tried to \
+				control you, but was unable to</span>"
+			qdel(M)
+
+			invoker << "<span class='sevtug'>Juvyr V fgvyy jnvg va gur \
+				Pryrfgvny Qreryvpg, zl graqevyf ner irel sentvyr. Fbzrguvat \
+				vagrecbfrq vgfrys orgjrra zr naq gur gnetrg, naq \
+				fnpevsvprq vgfrys.</span>\n\
```
Comment (Xhuis) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: Nitpick: Seems kind of unfitting of Sevtug, who's generally a jerk to everyone and doesn't really take the whole situation seriously. Perhaps "Gurl chg fbzr zrgny va gurve urnq gb cebgrpg gurz. Vf guvf gur yngrfg snfuvba gurfr qnlf?" or something like that would work better?

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 2]
```diff
@@ -1058,6 +1058,26 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 	if(!target)
 		return 0 //wait where'd they go
 	clockwork_generals_invoked["sevtug"] = world.time + CLOCKWORK_GENERAL_COOLDOWN
+
+	// You can throw off the mind control if you're shielded
+	for(var/obj/item/weapon/implant/mindshield/M in target)
+		if(M.implanted)
+			M << "<span class='sevtug'>Lbhe anabobgf jvyy abg cebgrpg lbh \
```
Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: Bu, ybbx, n zvaqfuvryq. Phgr, ohg V'yy uhzbe vg.

**Comment by coiax** [comment]
I defer to you, masters of clockcult.

**Comment by Kromgar** [comment]
>destroys implants

This makes it sooo much better. Now you just have to continually purchase mind shield implants so all the heads dont get possessed and taken to clock cult.

**Comment by MMMiracles** [comment]
Make it something like neural feedback from XCOM where a failed mind-control does a bunch of damage to the attempted mind-controller after breaking said implant.

The excuse being that nanotrasen put feedback responses in their implants to deter would-be zealots from fucking with their corporation.

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 3]
```diff
@@ -1058,6 +1058,25 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 	if(!target)
 		return 0 //wait where'd they go
 	clockwork_generals_invoked["sevtug"] = world.time + CLOCKWORK_GENERAL_COOLDOWN
+
+	// You can throw off the mind control if you're shielded
+	for(var/obj/item/weapon/implant/mindshield/M in target)
+		if(M.implanted)
+			M << "<span class='sevtug'>Bu, ybbx, n zvaqfuvryq. Phgr, ohg \
```
Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: Incidentally this is hilarious if you translate it sevtug is literally bullshitting the invoker

**Comment by Kromgar** [comment]
Mind control is stupid fucking op. You can convert anyone you want very easily. Instead people will just make mind control chains to get the high profile target.

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 4]
```diff
@@ -1058,6 +1058,25 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
...
+				V'yy uhzbe vg.</span>\n\
+				<span class='userdanger'>You feel like something tried to \
+				control you, but was unable to</span>"
```
Comment (PKPenguin321) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: punctuate.

File: code/game/gamemodes/clock_cult/clock_scripture.dm (hunk 3 above)
Comment (RemieRichards) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: remove ohg, ohg is unnecessary

Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: ohg discrimination

**Comment by MMMiracles** [comment]
Seriously though @Kromgar is pretty much right, without any kind of repercussion for trying to control an implanted person you'll just end up with people sitting there prioritizing targets as they stagger around to frantically get another implant before the next mind control attempt.

**Comment by ChangelingRain** [comment]
The repercussion is a 5 minute cooldown and all the component use

**Comment by MMMiracles** [comment]
Component use is hardly a repercussion when you can easily stockpile them with bases.

Forcing security to waste all their implants on 1-2 important people and wasting all of cargo's resources on further implants to prevent being thrown out the airlock against your will is pretty shitty.

File: code/game/gamemodes/clock_cult/clock_scripture.dm (hunk 3 above)
Comment (RemieRichards) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: also uhzbe should be uhzbhe

**Comment by coiax** [comment]
You realise that this is a nerf? Currently there is no defence vs Sevtug.

**Comment by ChangelingRain** [comment]
I really don't like mind control and would rather remove it tbh

**Pull request closed without merging**